src/open_inwoner/js/components/form/FileInput.js

```diff
@@ -137,22 +169,22 @@ export class FileInput extends Component {
 
   /**
    * Generic no op (no operation) event handler. Calls `preventDefault()` on given event.
-   * @param {Event} e
+   * @param {Event} e - The event.
    */
   noop(e) {
     e.preventDefault()
   }
 
   /**
    * Adds files in dataTransfer to input, only the first item is added if not `[multiple]`.
-   * @param {File[]} files
+   * @param {File[]} files - Array of files to add.
```

Comment:
Technically this is a `FileList`, not an Array (even though the jsdoc type annotation is incorrect here as well).